File: albedo-boot-ui-angular/src/main/webapp/app/theme/pages/modules/manage/group/group.service.ts
/**
 * Copyright &copy; 2018 <a href="https://github.com/somewhereMrli/albedo-boot">albedo-boot</a> All rights reserved.
 */
import { Injectable } from '@angular/core'
import { HttpClient } from "@angular/common/http";
import { CTX } from "../../../../../app.constants"
import { DataService } from "../../../../../shared/base/service/data.service";
import { PublicService } from "../../../../../shared/base/service/public.service";
import { Group } from "./group.model";


@Injectable()
export class GroupService extends DataService<Group> {

    constructor(
        protected http: HttpClient,
        protected publicService: PublicService) {
        super(http, publicService.getServiceCtx('manage_group') + '/manage/group')
    }

}

File: albedo-boot-api/albedo-boot-service-mybatis/src/main/java/com/albedo/java/modules/manage/service/GroupService.java
/**
 * Copyright &copy; 2018 <a href="https://github.com/somewhereMrli/albedo-boot">albedo-boot</a> All rights reserved.
 */
package com.albedo.java.modules.manage.service;

import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

import com.albedo.java.common.persistence.service.DataVoService;
import com.albedo.java.modules.manage.domain.Group;
import com.albedo.java.modules.manage.domain.vo.GroupVo;
import com.albedo.java.modules.manage.repository.GroupRepository;

/**
 * 分组信息管理Service 分组信息管理
 * @author admin
 * @version 2019-04-04
 */
@Service
@Transactional(rollbackFor = Exception.class)
public class GroupService extends DataVoService<GroupRepository, Group, String, GroupVo>{



}
